### ISSUE TITLE
get value from project's custom field and set issue custom field when creating issue

### ISSUE THREAD

**Issue by oamoris** [anteo/redmine_custom_workflows#157]
Hi!
I have issue custom field which is set of users (ex 'responsible'). And I have project custom field which is also set of users (ex 'responsibles default').
What I need: When creating issue take value of 'responsibles default' of current project and set its value to 'responsible' custom field of issue.
Is it possible?

**Comment by picman** [comment]
Yes, why not? Just copy the custom field from the project to the issue in before_save hook.

**Comment by frankz96** [comment]
@picman Hi!

I tried a similar question, i want to change a custom field (user) based on issue status, but it doesn't work, here is my code:

        if @issue.status_id == 2
          @issue.custom_field_value_2 = @assigned_to
        end

could you help me to make it work?
